components/StatusFilter.tsx

```diff
@@ -20,12 +20,12 @@ interface StatusAction {
 }
 
 interface StatusFilterProps {
-  handleChecks: (value: string) => void;
+  handleApplyStatusFilterCheck: (value: string) => void;
 }
-const reducer = (statusFilterStates: StatusFilter[], action: StatusAction) => {
+const reducer = (statusFilterState: StatusFilter[], action: StatusAction) => {
   switch (action.type) {
     case 'toggle':
-      return statusFilterStates.map((statusFilterState: StatusFilter) => {
+      return statusFilterState.map((statusFilterState: StatusFilter) => {
```

Comment:
이 경우에는 따로 타입을 잡아주지 않아도 괜찮아요~

```suggestion
      return statusFilterState.map((statusFilterState) => {
```